### DISCUSSION TITLE
I am confused about this project, do I need seperate esphome image?

### DISCUSSION THREAD

**Discussion by ForceConstant** [emporia-vue-local/esphome#247] [General]
So aside from the helpful  README, what is all the other software that is part of this project? The instructions in the README, don't ever actually refer to it, but do state you need "ESPHome image" do we get that as part of this project, or the base esphome project? And what is the dockerfile/image being used for? I would like to install using docker.

**Comment by flaviut** [comment]
That wording is confusing. You just need a working ESPHome installation, I've updated the wording to make it clear.

Once you have a [regular ESPHome working based on the official instructions](https://esphome.io/), you create a new device the regular way with the config in the README.
